dev.ts

```diff
@@ -238,9 +244,69 @@ const templates = {
     obj: (file: string, i: number) =>
       `${JSON.stringify(`./sections${file}`)}: $$$${i},`,
   },
-  loaders: {
+  functions: {
     imports: (file: string, i: number) =>
-      `import * as $$$$${i} from "./loaders${file}";`,
-    obj: (file: string, i: number) => `"./loaders${file}": $$$$${i},`,
+      `import * as $$$$${i} from "./functions${file}";`,
+    obj: (file: string, i: number) => `"./functions${file}": $$$$${i},`,
   },
 };
+
+/**
+ * Extracts JSONSChemas from types definitions in sections and functions.
+ *
+ * Read more about it here: TODO
+ *
+ */
+async function extractAllSchemas(
+  sections: string[],
+  functions: string[]
+): Promise<DevManifestData["schemas"]> {
+  const sectionSchemasAsArray = await Promise.all(
+    // TODO: Remove
+    sections.map(async (section) => {
+      const pathForSection = `./sections${section}`;
+
+      const denoDocJson = await readDenoDocJson(pathForSection);
+      const sectionSchema = getInputSchemaFromDocs(
+        denoDocJson,
+        `Section ${section}`
+      );
+
+      return {
+        key: pathForSection,
+        inputSchema: sectionSchema,
+        outputSchema: null,
+      };
+    })
+  );
+  const functionSchemasAsArray = await Promise.all(
+    functions.map(async (functionName) => {
+      const pathForFunction = `./functions${functionName}`;
+      const denoDocJson = await readDenoDocJson(pathForFunction);
+      const entityNameForLogging = `Function ${functionName}`;
+      const inputSchema = getInputSchemaFromDocs(denoDocJson);
+      const outputSchema = getFunctionOutputSchemaFromDocs(
+        denoDocJson,
+        entityNameForLogging
+      );
+
+      return { key: pathForFunction, inputSchema, outputSchema };
+    })
+  );
+  const schemas = [...sectionSchemasAsArray, ...functionSchemasAsArray]
+    .filter(
+      ({ inputSchema, outputSchema }) =>
+        Boolean(inputSchema) || Boolean(outputSchema)
+    )
+    .reduce((acc, cur) => {
+      return {
+        ...acc,
+        [cur.key]: {
+          inputSchema: cur.inputSchema as JSONSchema7,
+          outputSchema: cur.outputSchema as JSONSchema7,
+        },
+      };
```

Comment:
wdyt about this suggestion?:

```suggestion
      acc[cur.key]: {
        inputSchema: cur.inputSchema as JSONSchema7,
        outputSchema: cur.outputSchema as JSONSchema7,
      };
      return acc;
```